.gitignore

```diff
@@ -9,3 +9,7 @@ lua/src/luac
 cbenchmarks/*
 !cbenchmarks/*.c
 !cbenchmarks/makefile
+
+test.pln
+test.so
+test.lua
```

Comment:
We already have a gitignore rule to ignore ".so" files.

Reply:
I added these lines for development. I'll remove them before we merge.

Reply:
You could consider using `git add -u` instead of plain `git add`. That way it only adds changes to files that are already in the repository and ignores the extra test files that you create. No need to change the gitignore file.